src/Symfony/Bridge/Doctrine/Security/User/EntityUserProvider.php

```diff
@@ -107,6 +108,22 @@ public function supportsClass($class)
         return $class === $this->getClass() || is_subclass_of($class, $this->getClass());
     }
 
+    /**
+     * {@inheritdoc}
+     */
+    public function upgradePassword(UserInterface $user, string $newEncodedPassword): void
+    {
+        $class = $this->getClass();
+        if (!$user instanceof $class) {
+            throw new UnsupportedUserException(sprintf('Instances of "%s" are not supported.', \get_class($user)));
+        }
+
+        $repository = $this->getRepository();
+        if ($repository instanceof PasswordUpgraderInterface) {
+            $repository->upgradePassword($user, $newEncodedPassword);
```

Comment:
So the idea would be that users would make their UserRepository implement this, right? Hmm, I guess there’s no generic way to do this... I mean, we could have them configure a “password” field and use that to make the query for them... though I don’t love more yaml config. And.. we can’t make a trait generic enough to do this. I guess best I can think of us add this to the repository automatically in make:user. It’s also a bit unfortunate that (unless you use the maker) you could silently “miss out” on this feature by not implementing this interface. Again, this is more unhelpful pondering than solution :)

Reply:
Persistency is always custom, there is no single method we can provide to do it.
Adding requiring `setPassword()` on the entity wouldn't make it persist.
And yes, this must be opt-in, because not everyone will want/have the auth to *write* to the backend.
At this point, you need to review the maker-bundle PR:
https://github.com/symfony/maker-bundle/pull/389